Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM zenika/alpine-node:latest
```

Comment:
Why not use an official node alpine image?

Reply:
zenika/alpine-node looked like a well maintained, minimal alpine image, but I can switch over to the official node-alpine image, it's a small change that will just slightly increase build times.